web/src/site/FreeUsersExceededAlert.tsx

```diff
@@ -0,0 +1,23 @@
+import WarningIcon from 'mdi-react/WarningIcon'
+import * as React from 'react'
+
+/**
+ * A global alert that appears telling all users that they have exceeded the limit of free users allowed.
+ */
+export const FreeUsersExceededAlert: React.FunctionComponent<{
+    noLicenseWarningUserCount: number | null
+    className?: string
+}> = ({ noLicenseWarningUserCount, className = '' }) => (
+    <div className={`alert alert-danger alert-animated-bg align-items-center ${className}`}>
```

Comment:
```suggestion
    <div className={`alert alert-danger alert-animated-bg ${className}`}>
```
This only applies to flex boxes